library/python/BUILD

```diff
@@ -1,33 +1,20 @@
-load("@rules_python//python:defs.bzl", "py_binary")
 load("@pybind11_bazel//:build_defs.bzl", "pybind_extension", "pybind_library")
 
 licenses(["notice"])  # Apache 2
 
-py_binary(
-    name = "e2e_test",
-    srcs = ["e2e_test.py"],
-    data = [
-        ":envoy_engine.so",
-    ],
-    main = "e2e_test.py",
-    deps = [
-        "//library/python/gevent_util",
-    ],
-)
-
 pybind_library(
     name = "envoy_engine_lib",
     srcs = [
-        "bytes_view.cc",
-        "engine_builder_shim.cc",
-        "stream_prototype_shim.cc",
-        "stream_shim.cc",
+        "envoy_engine/bytes_view.cc",
```

Comment:
Generally we try to avoid prefixing targets/classes/files with "envoy" unless they represent a top-level interface. Any reason not to simply go with `engine/` here?

Reply:
>unless they represent a top-level interface

This might fall into that category, depending on its exact definition. These files collectively define the `envoy_engine.so` target, which could be imported without using `envoy_requests`. That said, there's no reason folder has to be named `envoy_engine`.

Reply:
Sorry, to clarify, let's make the folder be `engine` unless there's a reason not to go this way?